pylama.ini

```diff
@@ -226,3 +226,8 @@ ignore = R0201
 ignore = R0201
 
 # R0201 Method could be a function [pylint]
+
+[pylama:cloudmarker/events/azstorageaccountdefaultnetworkaccessevent.py]
+ignore = R0201
+
+# R0201 Method could be a function [pylint]
```

Comment:
Please add a newline at the end of this file.

Reply:
Addressed.